.gitignore

```diff
@@ -57,3 +57,9 @@ ramendev/build
 # e2e generated files
 /e2e/ramen-e2e
 /e2e/ramen-e2e.log
+
+# Invalid placeholder CRD files
+config/crd/bases/_.yaml
```

Comment:
Should be `/config/crd/bases/_.yaml` to make it clear that this should match only the root of the project.